examples/language-modeling/run_language_modeling.py

```diff
@@ -101,6 +102,13 @@ class DataTrainingArguments:
     mlm_probability: float = field(
         default=0.15, metadata={"help": "Ratio of tokens to mask for masked language modeling loss"}
     )
+    plm_probability: float = field(
+        default=0.15,
```

Comment:
I think default is set to 1/6 in the official code no? => maybe do the same here?